autogen/README.md

```diff
@@ -0,0 +1,86 @@
+# Autogen Framework Model Interfaces
+The Autogen framework gives 3 major file based interfaces for models: `model.json`, `modelparameters.m`, and `sm_stop_verify.m` and 1 major interface available as a struct in the Matlab workspace: `mp`.
+
+## model.json
+`model.json` is a required configuration file shared between the Autogen Framework and the Autogen Tools. This covers both system-level properties as well as run-time tunable parameters for the model via registers.  
+`model.json` properties:  
+- `system`: An object to describe the overall system  
+    - `target`: Which hardware system is being targeted, the `audiomini` or `audioblade` (default: `audiomini`)  
+    - `sampleClockFrequency`: The sampling rate of the audio being processed (default: `48000`)  
+    - `systemClockFrequency`: Clock rate of the audio processing on the FPGA (default: `98304000`)  
+    - `inputDataType`: Object representing the data type of the audio input  
+        - `wordLength`: number of total bits in the data type  
+        - `fractionLength`: number of fractional bits in the fixed point number  
+        - `signed`: true if the data type is signed, false otherwise  
+    - `outputDataType`: Object reprenting the data type of the audio output
+    (Note: Currently unused, only inputDataType is read)
+        - `wordLength`: number of total bits in the data type  
+        - `fractionLength`: number of fractional bits in the fixed point number  
+        - `signed`: true if the data type is signed, false otherwise  
+    - `numberOfChannels`: number of channels in the audio to be processed (default: `2`)  
+- `devices`: An array of objects representing devices. Simulation support is only for one device.  
+    - device object  
+        - `name`: Name of the Simulink model  
+        - `registers`: An array of objects representing run-time tunable parameters as registers  
+        - register object  
+            - `dataType`: object representing the data type of the register  
+                - `fractionLength`: number of fractional bits in the fixed point number  
+                - `signed`: true if the data type is signed, false otherwise  
+                - `type`: string representation of type ('boolean', 'string', 'int')  
+                - `wordLength`: number of total bits in the data type  
+            - `defaultValue`: default value to set the register to (should be 1 or 0 for boolean types)  
+            - `name`: name of the tunable parameter  
+            - `registerNumber`: offset in memory for the register (starts at 1 and must have registerNumber's defined between 1 and the highest value given)  
+
+## modelparameters.m
```

Comment:
Compared to "Verifying the Simulation", "modelparameters.m" doesn't feel like a very descriptive section title.

Reply:
I am not totally sure what to say there besides the filename. Possibly "Configure the Simulation" or "Configure Autogen Simulation parameters" with `model.json` getting "Configure Autogen". I am not sure thats overly clear what those 2 different titles mean given everything is still just "Autogen".

Reply:
Maybe just "Model Parameters"?